packages/shared-state/src/reducers/preferences/reducer/index.ts

```diff
@@ -14,7 +14,7 @@ const initialState: PreferencesState = {
     // TODO: use system as a default once the dark theme is available
     theme: "light",
     preferDecentralization: false,
-    ipfsGatewayURL: "https://gateway.carrot-kpi.dev",
+    ipfsGatewayURL: "https://gateway.api.carrot.community",
```

Comment:
I think we should use the new `getServiceURL` here, no?

Reply:
The thing is that this addess will be overridden anyway on the app's mount. It's a bit of a weird setup